### PR TITLE
Handoff

### PR DESCRIPTION
**Pull request by melindaloubser1**
Implement handoff to another bot/host in conjunction with [this PR for chatroom](https://github.com/RasaHQ/chatroom/pull/1) and [this PR for Financial-Demo](https://github.com/RasaHQ/financial-demo/pull/57)

### CODE REVIEW

**Comment by melindaloubser1** [comment]
@rgstephens this one and the one on financial-demo are super similar, where the readme is concerned it's identical - if you could keep comments on one PR that might be easiest?

**Comment by melindaloubser1** [comment]
It also looks like Github is having some issues so the very last commits don't show; I'll let you know when they do

File: chatroom_handoff.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<head>
+  <link rel="stylesheet" href="http://127.0.0.1:8080/dist/Chatroom.css" />
+</head>
+<body>
+  <div class="chat-container"></div>
+
+  <script src="http://127.0.0.1:8080/dist/Chatroom.js"/></script>
+  <script type="text/javascript">
+    var chatroom = new window.Chatroom({
+      host: "http://192.168.64.2/webhooks/rest/webhook",
```
Comment (rgstephens) [inline chatroom_handoff.html:10]: Should this point to localhost and is `handoffhost` still used?

Comment (melindaloubser1) [reply chatroom_handoff.html:10]: ooh no it should not be like this! I'll fix it

Comment (melindaloubser1) [reply chatroom_handoff.html]: Hmm, also realize we need more straightforward instructions for actually starting this

**Comment by melindaloubser1** [comment]
Ooh, we should probably merge finbot & chatroom too for this  to make sense together? We'll also need to set up chatroom on the servers for demos.

**Pull request merged**